components/FooterModal.tsx

```diff
@@ -61,8 +58,7 @@ export default function FooterModal({
         _hover={{}}
         fontSize={["xs", "xs", "xs", "sm"]}
         onClick={handleOnOpen}
-        rel="noopener noreferrer"
-        target="_blank"
+        isExternal
```

Comment:
😯 Did not know about this!

Reply:
Super useful especially if like me and never remember the `rel="noopener noreferrer"` :joy: